gwsurrogate/surrogate.py

```diff
@@ -1617,15 +1617,20 @@ def __call__(self, q, chiA0, chiB0, M=None, dist_mpc=None, f_low=None,
                 using times/freqs. Default None.
 
     ellMax:     Maximum ell index for modes to include. All available m
-                indicies for each ell will be included automatically.
-                Default: None, in which case all available modes wll be
-                included.
+                indicies for each ell will be included automatically. The 
+                m<0 modes will automatically be included for nonprecessing
+                models. 
+                Default: None, in which case all available modes will be
+                included. For nonprecessing, setting ellMax=None will
+                not return m<0 modes.
 
     mode_list : A list of (ell, m) modes tuples to be included.
                 Example: mode_list = [(2,2),(2,1)].
                 Default: None, in which case all available modes are included.
-                The m<0 modes will automatically be included for nonprecessing
-                models. At most one of ellMax and mode_list can be specified.
+                The m<0 modes will not be included for nonprecessing models; 
+                please use ellMax instead.
```

Comment:
This can be confusing, because m<0 will actually be included if the user asks for h. This makes it sounds like one should never use mode_list.

Reply:
Done!